### DISCUSSION TITLE
Using Dagre.js with Vue-flow to make dynamic layout

### DISCUSSION THREAD

**Discussion by Ulrich-Mbouna** [bcakmakoglu/vue-flow#772] [Q&A] [answered]
Hello,
I want to know if it's possible to use [dagre.js](https://github.com/dagrejs) to make dynamic layout of our graph in vue-flow.

Thanks 🙏

**Comment by bcakmakoglu** [comment, marked as answer]
[There's an example using dagre.js right here](https://github.com/bcakmakoglu/vue-flow/blob/master/examples/vite/src/Layouting/LayoutingExample.vue)

**Reply by 985563349** [reply]
After switching the layout of the example, the wiring display is incorrect. How can I correct it?
